source/server-selection/tests/server_selection/ReplicaSetWithPrimary/write/DeprioritizedSecondaryPreferred.yml

```diff
@@ -0,0 +1,28 @@
+topology_description:
+  type: ReplicaSetWithPrimary
+  servers:
+  - address: b:27017
```

Comment:
This test is incorrect - this should select the primary.  From the spec:

> Write operations
> If the topology type is ReplicaSetWithPrimary, only an available primary is suitable for write operations.
> If the topology type is ReplicaSetNoPrimary, no servers are suitable for write operations.

and the primary is available